include/grub/slr_table.h

```diff
@@ -73,6 +73,8 @@
 #define GRUB_SLR_ET_UEFI_MEMMAP		0x0005
 #define GRUB_SLR_ET_RAMDISK		0x0006
 #define GRUB_SLR_ET_TXT_OS2MLE		0x0010
+#define GRUB_SLR_ET_MODULE		0x0011
```

Comment:
I think the idea was to group entry types, and this would fit in the same group as earlier types (0x0000-0x000f). By the look of it, 0x001? is used for things specific to TXT (or vendor-specific in general).

---

This applies to existing entry types as well: I think the names aren't descriptive enough. GRUB can boot many different operating systems, and specification is supposedly bootloader-agnostic. `GRUB_SLR_ET_SETUP_DATA` by itself doesn't tell too much. IMHO we should consider renaming it to `GRUB_SLR_ET_LINUX_SETUP_DATA`, even though it makes the name longer, similarly for `BOOT_PARAMS` and `MODULE`. This is however an issue with the specification, so I don't expect an immediate action here (except for numbering mentioned above).

Reply:
https://github.com/TrenchBoot/grub/compare/87f124a95cc8acc45103d9de33df11e8240d1776..94c7f4f9d2f33d3bf503c6356f779261674cf351#diff-fa8593325ef57d162ad9d9472e2e6571ec4dba071acb7da6e07515bbe8658a63